cli/lib/kontena/cli/service_command.rb

```diff
@@ -31,10 +35,14 @@ class Kontena::Cli::ServiceCommand < Clamp::Command
   subcommand "containers", "List service containers", Kontena::Cli::Services::ContainersCommand
   subcommand "logs", "Show service logs", Kontena::Cli::Services::LogsCommand
   subcommand "stats", "Show service statistics", Kontena::Cli::Services::StatsCommand
+  subcommand "monitor", "Monitor", Kontena::Cli::Services::MonitorCommand
+
   subcommand "envs", "Show environment variables", Kontena::Cli::Services::EnvsCommand
   subcommand "add-env", "Add environment variable", Kontena::Cli::Services::AddEnvCommand
   subcommand "remove-env", "Remove environment variable", Kontena::Cli::Services::RemoveEnvCommand
-  subcommand "monitor", "Monitor", Kontena::Cli::Services::MonitorCommand
+
+  subcommand "add-secret", "Add secret from Vault", Kontena::Cli::Services::AddSecretCommand
```

Comment:
simple reason: we will refactor sub-commands to new (better) style in a one PR .. target for that is post 0.12